### DISCUSSION TITLE
/queue pass function error

### DISCUSSION THREAD

**Discussion by VanLyden** [c0nD/FluffyBOT#64] [Bug Report]
Hi guys...found another bug again lmao. I've attached the image below, but basically any time someone passes their turn in the queue, Fluffy says that the **next** person up in the queue passed **and** is up at the same time, when it should say the person that was pinged passed their turn. In the pic, Lepias passed her turn in the queue, but it said I passed and that I was up next.

<img width="721" alt="Screenshot 2023-03-11 at 4 58 20 PM" src="https://user-images.githubusercontent.com/115385250/224515189-4c21c741-9ae2-4587-a693-cfcaf0b5e558.png">

**Comment by c0nD** [comment]
Ahh thanks for the report -- turning into an issue now.

**Comment by c0nD** [comment]
Should be fixed with PR #66 

(Keeping thread open incase it pops back up.)

**Reply by c0nD** [reply]
Confirmed bug fixed.

Closed.
